makeframemd5

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env bash
+
+SCRIPTDIR=$(dirname "${0}")
+. "${SCRIPTDIR}/mmfunctions" || { echo "Missing '${SCRIPTDIR}/mmfunctions'. Exiting." ; exit 1 ;};
+
+_usage(){
+    echo
+    echo "$(basename "${0}") ${VERSION}"
+    echo "Produces framemd5s for video files in a target directory."
+    echo "Usage: $(basename "${0}") [ -h ] package1 [ package2 ... ]"
```

Comment:
Makes sense to me! But I think it might be best to leave it in this form for consistency with the majority of the other mm scripts.